nimare/meta/cbma/ale.py

```diff
@@ -117,22 +117,28 @@ def _compute_null_analytic(self, ma_maps):
             raise ValueError('Unsupported data type "{}"'.format(type(ma_maps)))
 
         # Determine bins for null distribution histogram
+        # Remember that numpy histogram bins are bin edges, not centers
+        # Assuming values of 0, .001, .002, etc., bins are -.0005-.0005, .0005-.001, etc.
+        inv_step_size = 100000
+        step_size = 1 / inv_step_size
         max_ma_values = np.max(ma_values, axis=1)
+        # round up based on resolution
+        max_ma_values = np.ceil(max_ma_values * inv_step_size) / inv_step_size
         max_poss_ale = self.compute_summarystat(max_ma_values)
-        step_size = 0.0001
-        hist_bins = np.round(np.arange(0, max_poss_ale + 0.001, step_size), 4)
-        self.null_distributions_["histogram_bins"] = hist_bins
-
-        ma_hists = np.zeros((ma_values.shape[0], hist_bins.shape[0]))
+        # create bin centers, then shift them into bin edges
+        hist_bins = np.round(np.arange(0, max_poss_ale + (1.5 * step_size), step_size), 5) - (
```

Comment:
The 1.5 _should_ result in the maximum possible statistic value falling within the last bin, with no trailing bins after that.